src/db.rs

```diff
@@ -47,7 +47,7 @@ impl DB {
         })
     }
 
-    pub fn open_old<P1, P2>(path_old: P1, path: P2) -> Result<DB>
+    pub fn open_and_migrate<P1, P2>(path_old: P1, path: P2) -> Result<DB>
```

Comment:
I renamed this to be more indicative of what it does -- it opens the old database and migrates it to the new, XDG-conforming location; it doesn't "just" open the old database.

Reply:
Fair enough!